### ISSUE TITLE
Pico Driller not working - tonum needs to support fixed point binary string literals

### ISSUE THREAD

**Issue by trickey445** [jtothebell/fake-08#20]
Sorry if there was a comment section I could not find but I just wanted to thank you for makeing this

As I Really Love Pico-8  can't wait to give this ago if/when my  3ds gets here my old one got stolen 2 years ago with my 2 favourite games at the time :( anyway I was wondering if U have any plans to 
Change the icon? Also some screenshots s whod be cool :) I'd love to see 
Pico driller running on this!

**Comment by jtothebell** [comment]
Hello, and thank you. There isn't really a comment section. I may post about it in a few more places after the rest of the features are more fleshed out. Right now I'm focusing on adding a test suite and fixing bugs I find along the way. After the plan is to make it memory compatible with Pico 8, and then update the lua implementation to use fixed point numbers (possibly using z8lua). I don't have any plans to change the icon, but I would be open to it. I just threw together the icon so it wasn't using the default one. 

I'd like to get pico driller running too. I verified that currently it is not working. It looks like it will take fixed point number support in the `tonum` method before pico driller will work property, possibly some other fixes too, but that was one thing I saw right away. I'm going to update the title of this ticket so I can remember to come back to this.
